dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -366,6 +368,21 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(String.class)
           .describedAs("principal/host");
 
+  private final OptionSpec<String> optionHiveSaslQop =
+      parser
+          .accepts(
+              OPT_HIVE_SASL_QOP,
+              "The QOP of the SASL connection between hadoop and the dumper. "
+                  + "Please check the 'hadoop.rpc.protection' configuration for your cluster. "
+                  + "Mapping between the 'hadoop.rpc.protection' value and SASL QOP:\n"
+                  + "authentication: auth\n"
+                  + "integrity: auth-int\n"
+                  + "privacy: auth-conf")
```

Comment:
Consider making a enum for internal values of `saslQop` and converting user input via `withValuesConvertedBy` to this enum to guard against unexpected values.